generator/AWSPSGeneratorLib/HelpMaterials/Examples/ELB2/Edit-ELB2LoadBalancerAttribute.xml

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$newAttribute = New-Object Amazon.ElasticLoadBalancingV2.Model.LoadBalancerAttribute
+$newAttribute.Key = 'deletion_protection.enabled'
+$newAttribute.Value = 'true'
+Edit-ELB2LoadBalancerAttribute -LoadBalancerArn 'arn:aws:elasticloadbalancing:us-east-1:123456789012:loadbalancer/app/test-alb/3651b4394dd9a24f' -Attribute $newAttribute -Region us-east-1
```

Comment:
Can be simplified to `-Attribute @{Key = 'deletion_protection.enabled'; Value = 'true'}`